solr/solrj/build.gradle

```diff
@@ -50,20 +41,15 @@ dependencies {
   implementation 'org.apache.httpcomponents:httpclient'
   implementation 'org.apache.httpcomponents:httpcore'
 
-  implementation('org.apache.zookeeper:zookeeper', {
-    exclude group: "org.apache.yetus", module: "audience-annotations"
-    exclude group: "log4j", module: "log4j"
-    exclude group: "org.slf4j", module: "slf4j-log4j12"
-  })
-  implementation ('org.apache.zookeeper:zookeeper-jute') {
-    exclude group: 'org.apache.yetus', module: 'audience-annotations'
-  }
-
   testImplementation project(':solr:test-framework')
   testImplementation project(':solr:core')
   testImplementation project(':solr:solrj')
   testRuntimeOnly project(':solr:modules:sql')
 
+  // ideally ZK centric tests move to solrj-zookeeper but sometimes we depend on ZK here anyway
```

Comment:
Worth opening a follow up JIRA to track this?

Reply:
Umm; I don't care enough any way.  I did call it out in the PR conversation.